src/test/unit/profile-data.test.js

```diff
@@ -436,34 +436,41 @@ describe('upgrades', function() {
       serializedOldProcessedProfile0
     );
     compareProcessedProfiles(upgradedProfile0, afterUpgradeReference);
+
```

Comment:
I snuck this change in. These were really hard to read while I was debugging this.